### DISCUSSION TITLE
Chia appears to have Downgraded Version after a Reboot (Pool tab missing)

### DISCUSSION THREAD

**Discussion by DendyDarzartar** [Chia-Network/chia-blockchain#7424] [Support]
Hi All,

I have the issue described in #3012 where my Chia shows my version as 0.0. It displayed this prior to updating to the newest build and continued to do so after.

I had initially attempted to update Chia using the upgrade steps here (https://github.com/Chia-Network/chia-blockchain/wiki/INSTALL#ubuntudebian) under the "To Update/Upgrade from previous version" section.

I did not receive any errors while doing so however upon launching Chia I noticed the pool tab was not present so I downloaded the newest version from https://www.chia.net/ and ran through the new install steps on the install section referenced above.

When I launched chia the next time I did see the pool tab and all appeared to be well (aside from it showing version 0.0) however after rebooting my PC and relaunching Chia it appears that it has loaded an old version as the pool tab is now missing.

Is it possible that there is some residual install of the old version that launched rather than the newest version?

How can I go about deleting ALL chia references from my Linux system so that I may start fresh?

If you need any log files from me, please let me know where I can look to find them as I am very new to this OS and still learning more each day.

Thank you in advance!

**Comment by DroptheHammer** [comment]
I followed this guide as well and I'm showing 1.1.6dev0 on my version no matter how many times i follow the guide and never receive errors.  What's up Chia?
